templates/base/kernel-ci-base.jinja2

```diff
@@ -1,4 +1,4 @@
-{% set extra_kernel_args = "console_msg_format=syslog" %}
+{% set _ = context.update({'extra_kernel_args': "console_msg_format=syslog"}) %}
```

Comment:
I don't think it's right to do this here, because it will happen for *all* jobs, and adding `extra_kernel_args` to the context is not the right way to append the command line for all jobs.

See I updated the original patch to update the context in the u-boot template itself.

Reply:
I see.  However in https://github.com/kernelci/kernelci-core/pull/108 the extra_kernel_args is also being defined in `kernel-ci-base.jinja2`.  I'll put a comment there, and close this PR.